modules/nw-ovn-kubernetes-migration-about.adoc

```diff
@@ -104,7 +104,7 @@ For more information on using an egress firewall in OVN-Kubernetes, see "Configu
 [id="egress-router-pods_{context}"]
 === Egress router pods
 
-OVN-Kubernetes does not support using egress router pods in {product-title} 4.6.
+OVN-Kubernetes does not support using egress router pods in {product-title} 4.7.
```

Comment:
@ahardin-rh, this is going in for 4.7, and I'll need to remove this line in its entirety.

Reply:
Thanks, @jboxman. I'm pulling it out.